content/react.md

```diff
@@ -0,0 +1,271 @@
+---
+title: React
+order: 8
+description: How to use React, the frontend rendering library with Meteor.
+---
+
+After reading this guide, you'll know:
+
+1. What React is, and why you would consider using it with Meteor.
```

Comment:
IDK, it's in the middle of the meteor guide, I'm not sure it reads well to pitch meteor